js/main.js

```diff
@@ -1 +1,71 @@
 /* Javascript goes here! */
+
+$(function(){
+   $('.results').attr('style', 'display: none');
+  $('form').submit(function(e){
+    e.preventDefault();
+    requestWeatherData();
+  });
+});
+
+function requestWeatherData() {
+
+  var x = $("form").serialize();
+  var urlx = 'http://api.openweathermap.org/data/2.5/weather?'+x;
+
+ $.ajax({
+    method: 'GET',
+    url: urlx,
+    //data: $("form").serialize(),// 'q=Austin&appid=1c53e15b8d17e80f85194c2079c3afe2',
+    dataType: 'json',
+    success: onSuccess,
+    error: onError
+ });
+
+ function clearValues(){
```

Comment:
You've cleared all of these inputs with a space. You can also just clear by reseting to `('')`.